### PR TITLE
New Option:Dictator use Command /exp to Expel instead of Voting 

### PR DESCRIPTION
**Pull request by hinhinarrrrrr**
Option let dictator can use the command /exp or /expel to expel someone instead of voting.
Dictator cant vote normally.This Option can make dictator can vote normally

### CODE REVIEW

**Comment by Tommy-XL** [comment]
I will merge this after the release of the first beta 2.1.0

**Comment by 0xDrMoe** [comment]
can we get conflicts fixed before merge?

**Review by NikoCat233 — APPROVED** [review]

**Pull request closed without merging**
